### PR TITLE
Task5

### LINKED ISSUE #8: 4, 5) Фильтрация данных таблицы + поиск
В зависимости от текущего значения фильтра данные в таблице должны меняться

### PR DESCRIPTION
**Pull request by Chackmoole**

### CODE REVIEW

File: src/components/UsersTable/UsersTable.jsx [hunk 1]
```diff
@@ -8,11 +8,26 @@ import {
...
+    setCurrentStatus(value);
+  };
+  const [searchText, setSearchText] = useState('');
+  const onSearchChange = (e) => {
+    setSearchText(e.target.value);
+  };
+
+  const isInclud = (searchValue, stringValue) => {
+    return stringValue.toLowerCase().indexOf(searchValue.toLowerCase()) !== -1;
+  };
```
Comment (NataliaIlina) [inline src/components/UsersTable/UsersTable.jsx]: isInclude

File: src/components/UsersTable/UsersTable.jsx [hunk 2]
```diff
@@ -26,15 +41,21 @@ const UserTable = () => {
...
+                return (
+                  <tr key={user.id}>
+                    <StyledCell>{user.firstName}</StyledCell>
+                    <StyledCell>{user.lastName}</StyledCell>
+                    <StyledCell>{statusTitle[user.status]}</StyledCell>
+                    <StyledCell>{user.mail}</StyledCell>
+                    <StyledCell>{user.registrationDate}</StyledCell>
+                  </tr>
+                );
             })}
```
Comment (NataliaIlina) [inline src/components/UsersTable/UsersTable.jsx]: давай перепишем на https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter

File: src/components/UsersTable/UsersTable.jsx (hunk 1 above)
Comment (NataliaIlina) [reply src/components/UsersTable/UsersTable.jsx]: есть более современный аналог indexOf 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes

Comment (Chackmoole) [reply src/components/UsersTable/UsersTable.jsx]: done

File: src/components/UsersTable/UsersTable.jsx (hunk 2 above)
Comment (Chackmoole) [reply src/components/UsersTable/UsersTable.jsx]: done

**Pull request merged**